network/global/base-identities/roles.tf

```diff
@@ -164,3 +138,101 @@ module "iam_assumable_role_grafana" {
 
   tags = local.tags
 }
+
+#
+# Assumable Role: AWSServiceRoleForOrganizations
+#
+module "iam_assumable_role_service_organizations" {
+  source = "github.com/binbashar/terraform-aws-iam.git//modules/iam-assumable-role?ref=v4.23.0"
+
+  trusted_role_services = [
+    "organizations.amazonaws.com"
+  ]
+
+  create_role      = true
+  role_name        = "AWSServiceRoleForOrganizations"
+  role_description = "Service-linked role used by AWS Organizations to enable integration of other AWS services with Organizations."
+  role_path        = "/aws-service-role/organizations.amazonaws.com/"
+
+  role_requires_mfa = false
+
+  custom_role_policy_arns = [
+    "arn:aws:iam::aws:policy/aws-service-role/AWSOrganizationsServiceTrustPolicy"
+  ]
+
+  tags = local.tags
+}
+
+#
+# Assumable Role: AWSServiceRoleForSupport
+#
+module "iam_assumable_role_service_support" {
+  source = "github.com/binbashar/terraform-aws-iam.git//modules/iam-assumable-role?ref=v4.23.0"
+
+  trusted_role_services = [
+    "support.amazonaws.com"
+  ]
+
+  create_role      = true
+  role_name        = "AWSServiceRoleForSupport"
+  role_description = "Enables resource access for AWS to provide billing, administrative and support services"
+  role_path        = "/aws-service-role/support.amazonaws.com/"
+
+  role_requires_mfa = false
+
+  custom_role_policy_arns = [
+    "arn:aws:iam::aws:policy/aws-service-role/AWSSupportServiceRolePolicy"
+  ]
+
+  tags = local.tags
+}
+
+#
+# Assumable Role Cross-Account: OrganizationAccountAccessRole
+#
+module "iam_assumable_role_oaar" {
+  source = "github.com/binbashar/terraform-aws-iam.git//modules/iam-assumable-role?ref=v4.1.0"
+
+  trusted_role_arns = [
+    "arn:aws:iam::${var.accounts.management.id}:root"
```

Comment:
@eze-godoy We've roll-back this change (https://github.com/binbashar/le-tf-infra-aws/pull/417) because it mess the leverage SSO setup for the management acocunt.

CC: @marianod92 @angelofenoglio

Reply:
```suggestion
    "arn:aws:iam::${var.accounts.root.id}:root"
```